Midstone/ComponentFramework/Constants.h

```diff
@@ -7,6 +7,8 @@ using namespace MATH;
 
 const Vec3 ORIGIN = Vec3(0, 0, 0);
 
+const float CAMERA_TO_ORIGIN = 20;
```

Comment:
The distance we have now (20) isn't constant and is just there for testing. just a heads up

Also this really shouldn't be constant as the player can move the camera